README.md

```diff
@@ -243,11 +243,17 @@ The use-case for composing multiple props mixins into a single component props c
 
 #### UiProps as a Map
 
+> [!WARNING]
```

Comment:
GitHub's "rich diff" viewer apparently doesn't render [alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) very well, so I'd recommend viewing this file directly: https://github.com/Workiva/over_react/blob/null-safety-docs/README.md